.github/workflows/main.yml

```
@@ -49,6 +49,14 @@ jobs:
 
       - run: python -m tox
 
+      # pytest-cov automatically merges coverage into `.coverage`, so we have to
+      # make it unique before uploading.
+      - name: Make coverage unique if it exists.
+        run: |
+          if [ -f .coverage ]; then
+            mv .coverage .coverage.$(uuidgen)
```

https://docs.github.com/en/actions/learn-github-actions/contexts#strategy-context suggests you use
```suggestion
            mv .coverage .coverage.${{ strategy.job-index }}
```
here